shell-ui/src/index.js

```diff
@@ -136,16 +174,29 @@ const SolutionsNavbar = ({
 
       return (
         <AuthProvider {...oidcConfig}>
-          <UserDataListener userGroupsMapping={config.userGroupsMapping} onAuthenticated={onAuthenticated} />
-          <StyledComponentsProvider
-            theme={{
-              // todo manages theme https://github.com/scality/metalk8s/issues/2545
-              brand: defaultTheme.dark,
-              logo_path: '/brand/assets/branding-dark.svg',
-            }}
-          >
-            <Navbar options={computedMenuOptions} userGroupsMapping={config.userGroupsMapping} />
-          </StyledComponentsProvider>
+          <LanguageProvider onLanguageChanged={onLanguageChanged}>
+            <ThemeProvider onThemeChanged={onThemeChanged}>
+              {(theme, themeName) => (
+                <>
+                  <UserDataListener
+                    userGroupsMapping={config.userGroupsMapping}
+                    onAuthenticated={onAuthenticated}
+                  />
+                  <StyledComponentsProvider
+                    theme={theme.brand}
+                  >
+                    <Navbar
+                      logo={logos[themeName] || config?.logo?.[themeName] || `/brand/assets/logo-${themeName}.svg`}
+                      canChangeLanguage={canChangeLanguage !== undefined && canChangeLanguage !== null && canChangeLanguage !== 'false' || config.canChangeLanguage}
+                      canChangeTheme={canChangeTheme !== undefined && canChangeTheme !== null && canChangeTheme !== 'false' || config.canChangeTheme}
```

Comment:
Wondering would it be the same as
```suggestion
                      canChangeTheme={canChangeTheme && canChangeTheme === 'true' || config.canChangeTheme}
```

Reply:
Sadly not, the goal here is that `canChangeTheme` prop should be prioritised if defined, so if it is `false`, canChangeTheme should be false.

Reply:
If I am not mistaken, prop`canChanegTheme` that from `<SolutionsNavbar/>` is a string. 
So if`canChanegTheme` is `'false'`, then the value will depend on  `config.canChangeTheme` in your code.